lib/gulp-image-converter.js

```diff
@@ -41,7 +42,29 @@ function convert(options, file, cb) {
   });
 }
 
+function optimizeSvg(svgo, file, cb) {
+  var data = file.contents.toString('utf8');
+  if (data.indexOf('&ns_') !== -1) {
```

Comment:
hey @toolness why are we checking `&ns_`?  I barely know anything about `SVG` and I don't know what to Google for :grimacing: